### ISSUE TITLE
Overview of config options

### ISSUE THREAD

**Issue by einarq** [millionco/react-doctor#85]
Thanks for the awesome tool!

Struggling to find the various config options. I could find the jsx-a11y options, but cannot easily what other rules I can disable. Certainly a skill issue on my end, but perhaps there could be a link to the various tool configs we can adjust on the README?

**Comment by aidenybai** [comment]
Config options are now fully documented in the README.
